### ISSUE TITLE
[A11y, iOS] focus issues related to ActivityIndicator

### ISSUE THREAD

**Issue by PureWeen** [xamarin/Xamarin.Forms#4204]
### Description

The activity indicator needs to be able to take and then return focus to a relevant component on the screen. 

The spec and behavior for this still needs to be fleshed out a bit more

### Steps to Reproduce

1.  run the attached project
2.  Click the button that loads the activity indicator
3. once the activity indicator is gone nothing is focused
4. in the code I tell it to focus the button they clicked but it doesn't work

The user shouldn't really even have to tell the button they clicked to refocus. Once the visibility of the activity indicator goes away the button should refocus

1.  run the attached project
2.  Click the button that pushes a page to load the activity indicator
3. once the activity indicator is gone nothing is focused
4. The button they clicked to push the page should have focus

### Expected Behavior
When using an activity indicator the focused control after it is done should be something or should be specifiable

### Actual Behavior
The ability to focus controls after using an activity indicator is buggy and unreliable

### Basic Information

- Version with issue: 3.4 nightly
- Platform Target Frameworks: 
  - iOS:  12
  - Android:  didn't test
  - UWP: didn't test

### Reproduction Link

[ActivityA11y.zip](https://github.com/xamarin/Xamarin.Forms/files/2511452/ActivityA11y.zip)
